src/Adapter/Configuration.php

```diff
@@ -85,6 +86,31 @@ public function add(array $parameters = array())
         }
     }
 
+    /**
+     * Gets translatable configuration values
+     *
+     * @param string $key
+     * @param bool $activeLanguage - if set to false, then it retrieves translations of the
+     * languages which are disabled
+     * @param null|int $idShopGroup
+     * @param null|int $idShop
+     *
+     * @return array - array key is the language id
+     */
+    public function getInternational($key, $activeLanguage = true, $idShopGroup = null, $idShop = null)
```

Comment:
I disagree with this: how about a new class called `MultiLangConfiguration` with `get` and `getIncludingInactiveLocales()` functions?

```php
namespace PrestaShop\PrestaShop\Adapter\Configuration;

use Configuration as ConfigurationLegacy;
use Language as LanguageLegacy;

class MultiLangConfiguration implements MultiLangConfigurationInterface
{
    public function __contruct(ConfigurationLegacy $configuration)
    {
        $this->configuration = $configuration;
    }

    /* ... */ put your updated methods here.
} 
```

> As a general rule, we forbid boolean flags on new methods.